### PR TITLE
switch from rodare to zenodo

### PR DESCRIPTION
**Pull request by psychocoderHPC**
Old releases were already published on zenodo so we keep zenodo instead of using HZDR rodare.
To keep  zenodo was requested by @ax3l [here](https://github.com/alpaka-group/mallocMC/pull/200#issuecomment-760559982).

This PR is against the release branch 2.5.0 and will be backported after we released.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "title": "mallocMC - Memory Allocator for Many Core Architectures",
+  "description": "This project provides a framework for fast memory managers on many core accelerators. It is based on alpaka to run on many different accelerators and implements the ScatterAlloc algorithm.",
+  "keywords": [
+    "mallocMC",
+    "CUDA",
+    "manycore",
+    "GPU"
```
Comment (bernhardmgruber) [inline .zenodo.json:8]: I would add `allocator` her as well.

**Review by bernhardmgruber — APPROVED** [review]
I tried to find a schema for these zenodo files, but I could not find one. The file is at least valid JSON. I trust you that it works. Otherwise just send another PR.

**Comment by psychocoderHPC** [comment]
> I tried to find a schema for these zenodo files, but I could not find one. The file is at least valid JSON. I trust you that it works. Otherwise just send another PR.

I also past it always into a validator. A rodare validator would be nice.

**Comment by ax3l** [comment]
@bernhardmgruber the json schema is the same _invenio_ schema

**Comment by bernhardmgruber** [comment]
Thank you @ax3l! It seems you can get the JSON schema there somewhere, but it seems there is no userfriendly online validator available. Anyway, thx!

**Comment by ax3l** [comment]
Yep, would like that too.
Maybe one can write a gh action that uses the invenio schema python package to validate it on changes or so.

**Comment by bussmann** [comment]
Just my 2 cents: Helmholtz is working on key performance indices for Software. Then your rodare software can be easily added into the reporting (which is A LOT of work). Integration Zenodo: Not so much. Btw: I think KPIs for Software are stupid!

**Comment by psychocoderHPC** [comment]
@bernhardmgruber ready to merge

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    {
+      "name":"Worpitz, Benjamin"
+    },
+    {
+      "name":"Grund, Alexander",
+      "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid":"0000-0002-7196-8452"
+    }
+  ]
+}
```
Comment (sbastrakov) [inline .zenodo.json:51]: @psychocoderHPC is this missing new line important?

Comment (psychocoderHPC) [reply .zenodo.json:51]: I will add it to be sure that it is not making trouble

**Review by sbastrakov — APPROVED** [review]

**Pull request merged**